bundles/EcommerceFrameworkBundle/VoucherService/Token/Listing.php

```diff
@@ -67,11 +67,11 @@ public function setFilterConditions($seriesId, $filter = [])
                 $this->addConditionParam('length = ?', $filter['length']);
             }
 
-            if (isset($filter['creation_from'])) {
+            if (isset($filter['creation_from']) && $filter['creation_from'] !== '') {
```

Comment:
```suggestion
            if (!empty($filter['creation_from'])) {
```
That should be enough as well, right?

Reply:
Yes, it should - the additional check was to solve the problem like already existing line 62:

`if (isset($filter['usages']) && $filter['usages'] !== '') {`